airbyte-integrations/connectors/source-hubspot/sample_files/incremental_catalog.json

```diff
@@ -154,19 +160,13 @@
       "stream": {
         "name": "forms",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"]
-      },
-      "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite"
-    },
-    {
-      "stream": {
-        "name": "form_submissions",
-        "json_schema": {},
-        "supported_sync_modes": ["full_refresh"]
```

Comment:
Why not include this stream in the incremental catalog example/acceptance test?

Reply:
this stream is empty, while for incremental tests, all streams should produce at least 1 record

Reply:
And we don't have `empty_streams` for incremental tests, got it. Thanks for the explanation!